package.json

```diff
@@ -11,12 +11,14 @@
     "watch:all": "webpack -w",
     "flow": "flow",
     "gen-snapshot-tests": "node utils/gen-snapshot-tests.js && prettier --write packages/*/generated-snapshot.test.js",
+    "gen-styleguide-prod-config": "node utils/gen-styleguide-prod-config.js && prettier --write styleguide.prod.config.js",
     "jest": "SNAPSHOT_INLINE_APHRODITE=1 jest",
     "test": "yarn run lint && yarn run gen-snapshot-tests && yarn run build:all && yarn run flow && yarn run jest",
     "coverage": "SNAPSHOT_INLINE_APHRODITE=1 jest --coverage",
     "styleguidist": "styleguidist server",
     "start": "yarn run styleguidist",
-    "build:styleguidist": "styleguidist build",
+    "start:prod": "yarn run build:styleguidist && yarn run styleguidist --config styleguide.prod.config.js",
+    "build:styleguidist": "yarn run gen-styleguide-prod-config && styleguidist build --config styleguide.prod.config.js",
```

Comment:
Updated task to build the docs. Now starts with the `gen-styleguide-prod-config` task, then does the build using the transformed config instead of the original.

Reply:
🛑 So I do have a worry: this means that we can no longer see changes to prototype components when we do a PR to Github. Unfortunately, the same commands are used to build both the PR and the final release so I'm not sure that adding an extra command will help. It's just a bummer as we can no longer share prototypes with each other for testing purposes.